hles/extraction/src/main/scala/org/broadinstitute/monster/dap/HLESurveyExtractionPipeline.scala

```diff
@@ -9,6 +9,8 @@ import Args._
 object HLESurveyExtractionPipeline extends ScioApp[Args] {
 
   /** Names of all forms we want to extract as part of HLE ingest. */
+  val HLESEpoch = "2018-01-01"
```

Comment:
Just curious about this date - is this the earliest possible DAP pack date?

Reply:
A very good question, this was me being paranoid and choosing what I thought was an "early enough" date. I just pulled down the DAP Pack report someone had setup in RC and noted that the earliest date is 2019-12-23 so I will change to that.